src/test/ui/parser/unicode-chars.stderr

```diff
@@ -8,5 +8,11 @@ help: Unicode character ';' (Greek Question Mark) looks like ';' (Semicolon), b
 LL |     let y = 0;
    |              ^
 
-error: aborting due to previous error
+error: expected one of `.`, `;`, `?`, or an operator, found `;`
+  --> $DIR/unicode-chars.rs:2:14
+   |
+LL |     let y = 0;
+   |              ^ expected one of `.`, `;`, `?`, or an operator here
```

Comment:
Why doesn't parser see this token as a usual semicolon?
From the error above it's visible that lexer is able to recover is as a semicolon.

Reply:
I don't know, but it fixed itself after conversion to whitespace.